packages/commonwealth/server/migrations/20231121231746-community-url-updates.js

```diff
@@ -0,0 +1,110 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.sequelize.query(
+      `
```

Comment:
+1 on this -- will review once this is addressed.

Reply:
@kurtisassad Done. I used several BEGINs and COMMITs - not sure if it is overkill.

Reply:
Actually, I meant to use the sequelize transaction. One sec..

Reply:
Now using sequelize transaction. Thanks, good catch @kurtisassad

Reply:
@mhagel I'm not sure this migration is doing what you want it to. You have multiple SQL statements within a single `await queryInterface.sequelize.query()` call. From my understanding, this will only execute the first statement in the string (so none of the updates or deletions would occur). Each of the statements needs its own `await queryInterface.sequelize.query()` call with the transaction object included in the options.

Reply:
@timolegros It is working for me locally, but I can do them all separately if it is a better practice.

Reply:
@timolegros It will run fine. It needs to wait for the whole thing because the return value is governed by the last executed statement.

Reply:
My mistake. TIL `.query()` calls can pack together multiple statements. It seems to work and it seems to be properly transactionalized as well (tested by adding incorrect query in the middle of the statements and ensuring revert was successful).